src/WebJobs.Script/Config/RetryOptionsSetup.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Host.Config;
+using Microsoft.Azure.WebJobs.Script.Configuration;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.Config
+{
+    internal class RetryOptionsSetup : IConfigureOptions<RetryOptions>
```

Comment:
I didn't see a test for this options setup verifying that it correctly translates json into a RetryOptions